<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="passwordTxt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>  password</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="loginBtn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registBtn">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="banner">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Welcome to Payroll System</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameTxt">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>  Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Error: Username &amp; password not exists!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
